tests/kafkatest/services/trogdor/trogdor.py

```diff
@@ -316,6 +316,22 @@ def done(self):
             return task_state["state"] == TrogdorTask.DONE_STATE
         raise RuntimeError("Failed to gracefully stop %s: got task error: %s" % (self.id, error))
 
+    def running(self):
```

Comment:
It would be good to share some code between this and the done function.  It seems like they are 99% the same

Reply:
Sure, will do